src/docker/Dockerfile

```diff
@@ -52,43 +52,31 @@ RUN apt-get update && apt-get install -y \
     automake autoconf
```

Comment:
@amihaiemil majority of these things are not needed, please compare with https://github.com/yegor256/dockers/blob/master/xjm/Dockerfile and check locally if anything is missing